src/screens/lib/navigationService.ts

```diff
@@ -0,0 +1,31 @@
+import { createSecureStorage } from '@shared/lib';
+
+import { InitialRoute } from './types';
+
+const storage = createSecureStorage('navigation-storage');
+
+function SessionService() {
+  return {
+    setInitialRoute(route: InitialRoute) {
```

Comment:
Can we rename it to setStartupRoute?

Reply:
Not exactly. "Initial route" has been coined by react-navigation so I just follow their API naming